content/docs/user-guide/contributing/docs.md

```diff
@@ -46,6 +46,29 @@ Otherwise, please refer to the following procedure:
 
 We will review your PR as soon as possible. Thank you for contributing!
 
+## Installation
+
+Make sure you have the latest LTS version of [Node.js](https://nodejs.org) and
+[Yarn](https://yarnpkg.com) installed.
+
+Run `yarn`.
+
+## Commands
+
+- `yarn develop` - run dev server with hot reload.
+- `yarn build` - build static assets to `public` folder.
+- `yarn serve` - run static server over the `public` folder content to check
+  build results.
+- `yarn lint-ts` - lint `.ts` and `.tsx` for compilance with code style and
+  check its for type errors.
+- `yarn lint-css` - lint `.css` files for compilance with code style.
```

Comment:
Some commands are missing from this list. Maybe some of them (including build and serve) should stay in README and just link to that section of README from here?

Reply:
Maybe some of them (including build and serve) should stay in README and just link to that section of README from here?